### ISSUE TITLE
Segmentation fault : subscribe/publish v16.0.0

### ISSUE THREAD

**Issue by FlorianBELLAZOUZ** [uNetworking/uWebSockets.js#203]
Hi Alex!

I find Segmentation fault (core dumped) error by using subscribe/publish paterns on v16.0.0
I try to reduce the problem as little I can : 
````js
const Uws = require('uWebSockets.js')
const app = Uws.App()
app.ws('/*',{
  open:ws=>{
    ws.subscribe('+/red/+')
    ws.publish('one/red/a','one/red/a')
    ws.publish('two/red','two/red') } })
app.listen(8080,token=>token&&console.log('open 8080'))
````
I get the same error on SSL application.

OS: Linux 4.15.0-54-generic Ubuntu
Node.js: v11.15.0
Uws: 16.0.0

Don't hesitate to ask me for more information if you don't have enough to reproduce the error.

**Comment by dioptre** [comment]
Same here.

**Comment by alexhultman** [comment]
Thanks for the report. This is fixed in v16.1.0.

Also, there is now app.publish.
